assets/l10n/app_en.arb

```diff
@@ -194,6 +194,21 @@
       "filename": {"type": "String", "example": "file.txt"}
     }
   },
+  "messageListGroupUnknownUser": "(unknown user)",
+  "@messageListGroupUnknownUser": {
+    "description": "Name placeholder to use for user when we don't know their name."
+  },
+  "messageListGroupYouAndOthers": "You and {others}",
+  "@messageListGroupYouAndOthers": {
+    "description": "Message list sticky header for a dm group with others.",
```

Comment:
```suggestion
    "description": "Message list recipient header for a DM group with others.",
```

"DM" is an initialism, so in normal English text (as opposed to identifiers in code) it's all caps.

Also although these headers are sticky, I think "recipient header" works better as a name for them because it points more to what they're about.